3506/3/app/controllers/users_controller.rb

```diff
@@ -0,0 +1,49 @@
+class UsersController < ApplicationController
+  configure do
+    set :views, 'app/views'
+  end
+
+  get '/registration' do
+    erb :registration
+  end
+
+  get '/login' do
+    erb :login
+  end
+
+  post '/registration' do
+    @user = User.create(user_params)
```

Comment:
User won't be created if it invalid. You will call method `valid?` on nil in that case.